docs/Using-Fleet/MDM-migration-guide.md

```diff
@@ -98,7 +93,7 @@ Want to know what your organization can see? Read about [transparency](https://f
 
 2. On your **My device** page, follow the disk encryption instructions in the yellow banner. If you don’t see the **Turn on MDM** button, select the purple **Refetch** button at the top of the page. If you still don't see the **Turn on MDM** button after a couple minutes, please contact your IT administrator. If the **My device page** presents you with an error, please contact your IT administrator.
```

Comment:
This text should be about turning on filevault, rather than turning on MDM, but before I do that, I realized that we may be missing some states? Like what was the state that Rachael reported? https://fleetdm.slack.com/archives/C019FNQPA23/p1680645009610739?thread_ts=1680638583.735539&cid=C019FNQPA23

Reply:
Following up on Rachael's reported state in that Slack thread: https://fleetdm.slack.com/archives/C019FNQPA23/p1680703647396059?thread_ts=1680638583.735539&cid=C019FNQPA23